com.unity.cinemachine/Samples~/3D Samples/RoadieRun.unity

```diff
@@ -38,21 +38,21 @@ RenderSettings:
   m_ReflectionIntensity: 1
```

Comment:
I think you should add a custom blends asset to this scene, and make the walk->run blend faster than the run->walk blend.

Also consider tuning the camera a little, for the camera angles and because the zolly effect is a little extreme, I think.  We should try to make this look really nice, like something you would actually want in a game.